tests-behat/dropdown.feature

```diff
@@ -30,11 +30,16 @@ Feature: Dropdown
     When I press button "Save"
     Then Modal is open with text "match init: 1"
     When I click close modal
-    When I select "uTitle <b>\"' &lt;" in lookup "dropdown_single"
-    When I select "uTitle <b>\"' &lt;" in lookup "dropdown_single2"
-    When I select "uTitle <b>\"' &lt;" in lookup "dropdown_multi"
-    When I select "uTitle <b>\"' &lt;" in lookup "dropdown_multi2"
-    When I select "uTitle <b>\"' &lt;" in lookup "dropdown_multi_json"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "dropdown_single"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "dropdown_single2"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "dropdown_multi"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "dropdown_multi2"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "dropdown_multi_json"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "lookup_single"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "lookup_single2"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "lookup_multi"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "lookup_multi2"
+    When I select "uTitle <b>\"' &lt;&quot;&amp;" in lookup "lookup_multi_json"
     When I press button "Save"
     Then Modal is open with text "match u add: 1"
```

Comment:
All complex tests pass, thank you @lubber-de for going the jorney with me and @mkrecek234 for reporting the issue!

Reply:
@mvorisek Thanks a lot for sorting this out!